docs/examples/PanelCollapsible.js

```diff
@@ -12,7 +12,11 @@ class Example extends React.Component {
         <Button onClick={ ()=> this.setState({ open: !this.state.open })}>
           click
         </Button>
-        <Panel collapsible expanded={this.state.open}>
+        <Panel
+          collapsible
+          id="collapsible-panel-example"
+          expanded={this.state.open}
+        >
           Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
```

Comment:
it seems like there's a bit of branch-y special-casing just dealing with handling the case of omitting `<Panel.Body>`. do you think it'd be a huge loss just to always require `<Panel.Body>`? modals sort of work in this way of always requiring `<Modal.Body>`, no?